data/remnant/remnant.txt

```diff
@@ -276,6 +276,490 @@ phrase "remnant"
+		"your enemy's shields"
+		"the target's engines"
+	word
+		"."
+		"!"
+
+phrase "remnant salvage scanner"
+	word
+		"one or two"
+		"a dozen"
+		"some"
+	word
+		" "
+	word
+		"salvage scanners."
+	word
+		" "
+	word
+		"They combine outfit and tactical scanners into one package"
+		"Compared to your separated versions, our special scanner is more powerful"
+		"They have a longer range than your outfit scanner"
+		"Those babies scan fast"
+		"They're a must-have for plundering"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"Have you heard"
+		"Do you know"
+		"You know anything"
+	word
+		" "
+	word
+		"about"
+	word
+		" "
+	word
+		"the"
+		"those"
+	word
+		" "
+	word
+		"Quarg"
+		"Pug"
+		"Hai"
+		"Drak"
+		"Archons"
+		"insects far west of our space"
+	word
+		"?"
+	word
+		" "
+	phrase
+		"friendly remnant (aliens.!)"
+		"friendly remnant (aliens?)"
+
+phrase "friendly remnant (aliens.!)"
+	word
+		"They look strong"
+		"They look weak"
+		"Maybe we should contact them"
+		"I think they're dangerous"
+		"I think they're good"
+		"I think we can trust them"
+		"I want to plunder their tech"
+		"I want their ships"
+		"I haven't seen them yet"
+		"I wonder if the prefects already knew about them"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant (aliens?)"
+	word
+		"Are they strong"
+		"Are they weak"
+		"Are they friendly"
+		"Are they hostile"
+		"How strong are they"
+		"What are the odds of capturing their ships"
+		"Has your government met them"
+		"Is it worth trying to get my hands on their tech?"
+	word
+		"?"
+
+phrase "friendly remnant"
+	word
+		"Be careful"
+		"Stay alert"
+		"Be vigilant"
+	word
+		" "
+	word
+		"about"
+	word
+		" "
+	word
+		"the"
+	word
+		"Alphas in this galaxy"
+		"pirates in your space"
+		"Korath raiders"
+		"Korath"
+		"Archon in Nenia"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"You don't want to get yourself killed"
+		"It's dangerous to encounter them"
+		"Always have some weapons in case you come across them"
+		"We have ships that can cloak in case you find yourself in a tricky situation with them"
+	word
+		"."
+		"!"
+
+phrase "hostile remnant"
+	word
+		"(It consists of signs so complex that any regular human wouldn't understand, but it seems that they don't like you.)"
+
+phrase "hostile remnant"
+	word
+		"Apparently,"
+		"It seems"
+		"I think"
+	word
+		" "
+	word
+		"you're"
+	word
+		"not all that different from"
+		"very similar to"
+		"like"
+	word
+		"them"
+		"an Alpha"
+		"a certain genetically modified weapon I've heard about"
+		"a Korath"
+	word
+		"."
+	word
+		" "
+	word
+		"Too bad"
+		"Too sad"
+		"I hope you go up in flames"
+		"You backstabbed us"
+		"You lied to us"
+		"You betrayed our trust"
+	word
+		"."
+
+phrase "hostile remnant (we're)"
+	word
+		"we were"
+	word
+		" "
+	word
+		"friends"
+		"comrades"
+		"brethren"
+		"good"
+		"cool"
+
+phrase "hostile remnant (you're)"
+	word
+		"you were"
+	word
+		" "
+	word
+		"weren't like the Alphas"
```

Comment:
This can lead to "you were weren't like the Alphas." Use "not" instead of "weren't."
```suggestion
		"not like the Alphas"
```